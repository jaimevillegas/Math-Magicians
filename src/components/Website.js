// eslint-disable-next-line import/no-extraneous-dependencies
import { Outlet } from 'react-router-dom';

export default function Website() {
  return (
    <>
      <header className="header-website">
        <h2>Math Magicians</h2>
        <nav>
          <ul>
            <li><a href="/">Home</a></li>
            <li><a href="calculator">Calculator</a></li>
            <li><a href="quote">Quote</a></li>
          </ul>
        </nav>
      </header>
      <Outlet />
    </>
  );
}
