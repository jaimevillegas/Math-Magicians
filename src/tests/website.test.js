import { render, screen } from '@testing-library/react';
import Website from '../components/Website';

describe('Website test', () => {
  test('Website component is displayed', () => {
    render(<Website />);
    const website = screen.getByTitle('header-website');
    expect(website).toMatchSnapshot();
  });
});
