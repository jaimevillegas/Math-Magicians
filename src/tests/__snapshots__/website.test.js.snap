// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Website test Website component is displayed 1`] = `
<header
  class="header-website"
  title="header-website"
>
  <h2>
    Math Magicians
  </h2>
  <nav>
    <ul>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
